src/Traits/AdapterHelper.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\Shopware6Common\Traits;
+
+use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Shopware6Common\Export\Config\PluginConfig;
+use FINDOLOGIC\Shopware6Common\Export\Utils\Utils;
+use Vin\ShopwareSdk\Data\Entity\PropertyGroupOption\PropertyGroupOptionCollection;
+use Vin\ShopwareSdk\Data\Entity\PropertyGroupOption\PropertyGroupOptionEntity;
+
+trait AdapterHelper
+{
+    /**
+     * @return Attribute[]
+     */
+    public static function getPropertyGroupOptionAttributes(
+        PropertyGroupOptionCollection $collection,
+        PluginConfig $pluginConfig,
+    ): array {
+        $attributes = [];
+
+        foreach ($collection as $propertyGroupOptionEntity) {
+            $group = $propertyGroupOptionEntity->group;
+            if ($group && !$group->filterable) {
+                continue;
+            }
+
+            $attributes = array_merge($attributes, static::getAttributePropertyAsAttribute(
+                $propertyGroupOptionEntity,
+                $pluginConfig,
+            ));
+        }
+
+        return $attributes;
+    }
+
+    /**
+     * @return Attribute[]
+     */
+    protected static function getAttributePropertyAsAttribute(
+        PropertyGroupOptionEntity $propertyGroupOptionEntity,
+        PluginConfig $pluginConfig,
```

Comment:
You can also use member variables within traits, no need to provide them as parameter